### PR TITLE
README file update

### PR DESCRIPTION
**Pull request by forGGe**
/cc @vadimol

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,10 +36,7 @@ git clone https://github.com/forGGe/theCore.git
 When the code is received, you have to download and install all the dependencies.
 The process can take some time.
 To make it easier, I’d strongly recommend to download and install `nix`.
-The installation guide could be found
-[on the official website](https://nixos.org/nixos/manual/index.html#ch-installation) or
-[Domenkozar's blog](https://www.domenkozar.com/2014/01/02/getting-started-with-nix-package-manager/)
-.
+The installation guide could be found [on the official website](https://nixos.org/nixos/manual/index.html#ch-installation) or [Domenkozar's blog](https://www.domenkozar.com/2014/01/02/getting-started-with-nix-package-manager/).
```
Comment (vadimol) [inline README.md]: can be found?

**Comment by vadimol** [comment]
Reviewed. Added small note

File: README.md (hunk 1 above)
Comment (forGGe) [inline README.md]: !fixed

**Comment by vadimol** [comment]
looks good for me

**Pull request merged**